app/models/ci/commit.rb

```diff
@@ -125,10 +125,27 @@ def latest_statuses
       @latest_statuses ||= statuses.latest.to_a
     end
 
+    def latest_generic_statuses
```

Comment:
We should introduce method for CommitStatus to indicate whether the it should be shown on MR page.